backend/tests/unit/dal/Services/ProjectServiceTest.cs

```diff
@@ -600,11 +601,12 @@ public void Update_AsAdmin()
             // Act
             var projectToUpdate = service.Get(project.ProjectNumber);
             projectToUpdate.Description = "A new description";
-            var result = service.Update(projectToUpdate);
+            service.Update(projectToUpdate);
+            var result = service.Get(project.ProjectNumber);
```

Comment:
Oook, so it looks like returning the entity passed in from the controller is having a negative effect on our existing tests. Essentially, this is what is happening in service methods following this pattern.
1) The test creates a new Project (or parcel or whatever).
2) The object from #1 is used to update the database.
3) The object from #1 is returned, and the tests assert that object #1 is equal to object#1.

The problem is that the assertions in step 3 don't actually test anything. What we want to be testing is that the update function affects the entity within the database, what we are actually testing is that we are getting the same object reference out of an Update method that we put in.

As an example, the current dev version of the Update_UpdateBuilding test still passes if I change the ProjectService.Update method to just be:
```
public Project Update(Project project)
{
    return project;
}
```

In addition to testing issues, the object returned from this (and other) Update Api controllers cannot be trusted, as the data returned may not actually have been persisted. I haven't solved the issue with the API in this PR, but I have changed the tests to call update() and then call get() to retrieve the actual entity from the DB. However, it may make more sense to fix the issue in the API itself.